topics/robotics/index.md

```diff
@@ -0,0 +1,9 @@
+---
+aliases: robot
```

Comment:
It looks like the `robot` topic is used quite a bit for repositories that don't necessarily fall into this `robotics` realm. What are your thoughts on removing this alias?

Reply:
Good idea, removed!